packages/data-stores/src/starter-designs-queries/use-starter-designs-query.ts

```diff
@@ -0,0 +1,93 @@
+import { stringify } from 'qs';
+import { useQuery, UseQueryResult, QueryOptions } from 'react-query';
+import wpcomRequest from 'wpcom-proxy-request';
+import type {
+	StarterDesignsGenerated,
+	StarterDesignsGeneratedQueryParams,
+	AllStarterDesigns,
+} from './types';
+import type { Category, Design, DesignRecipe } from '@automattic/design-picker/src/types';
+
+interface Options extends QueryOptions< AllStarterDesignsResponse, unknown > {
+	enabled?: boolean;
+}
+
+interface AllStarterDesignsResponse {
+	generated: { designs: StarterDesignsGenerated[] };
+	static: { designs: StarterDesignStatic[] };
+}
+
+interface StarterDesignStatic {
```

Comment:
Nit: it sounds awkward -- maybe just `StaticDesign`? 😄 (anway they are private anyway though)

Reply:
Nit: Also, it is a bit confusing that the type for static designs is in this file but for the generated designs are in some other file... maybe (similar to my other comment) it is better to duplicate the type for `GeneratedDesign` here in this file? What do you think?